src/codeu/chat/client/commandline/Chat.java

```diff
@@ -228,7 +243,26 @@ private UserContext findUser(String name) {
         return null;
       }
     });
-
+    
+    // info (Server info)
+    //
+    // Get some infomation from server; it should be version info currently
+    //
+    panel.register("info", new Panel.Command() {
```

Comment:
This block of code now appears in this class twice - when you merged in master, other changes that implemented the "info" command were added (see lines 152-165 in this class). You should delete this block of code.